packages/docs-app/src/blueprint.md

```diff
@@ -30,71 +30,71 @@ The `main` module exports all symbols from all modules so you don't have to impo
 @### NPM installation
 
 1. Install the core package with an NPM client like `npm` or `yarn`, pulling in all relevant
-dependencies:
+   dependencies:
 
-  ```sh
-  yarn add @blueprintjs/core
-  ```
+    ```sh
+    yarn add @blueprintjs/core
+    ```
 
 1. If you see `UNMET PEER DEPENDENCY` errors, you should manually install React:
 
-  ```sh
-  yarn add react react-dom react-transition-group
-  ```
+    ```sh
+    yarn add react react-dom react-transition-group
+    ```
 
 1. Note that since the minimum supported version of React is [v16](https://reactjs.org/blog/2017/09/26/react-v16.0.html),
-all of its [JavaScript Environment Requirements](https://reactjs.org/docs/javascript-environment-requirements.html) apply to
-Blueprint as well. Blueprint components require the following ES2015 features:
+   all of its [JavaScript Environment Requirements](https://reactjs.org/docs/javascript-environment-requirements.html) apply to
+   Blueprint as well. Blueprint components require the following ES2015 features:
 
-  - `Map`
-  - `Set`
-  - `Array.fill`
-  - `Array.from`
+    * `Map`
+    * `Set`
+    * `Array.fill`
+    * `Array.from`
 
-  We recommend polyfilling these features using [es6-shim](https://github.com/paulmillr/es6-shim) or
-  [core-js](https://github.com/zloirock/core-js).
+    We recommend polyfilling these features using [es6-shim](https://github.com/paulmillr/es6-shim) or
```

Comment:
you went too far! i think it should be just three spaces like the paragraph on line 46.

run `yarn docs-data` in the root to regen docs site while you've got dev server running.

Reply:
that's prettier doing its thing. only 4 spaces increment